pyvista/core/filters/uniform_grid.py

```diff
@@ -414,3 +415,291 @@ def image_threshold(
+        """Apply the reverse fast fourier transform (RFFT) to the active scalars.
+
+        The input can have real or complex data in any components and data
+        types, but the output is always complex doubles, with the first
+        component containing real values and the second component containing
+        imaginary values. The filter is fastest for images that have power of
+        two sizes.
+
+        The filter uses a butterfly diagram for each prime factor of the
+        dimension. This makes images with prime number dimensions (i.e. 17x17)
+        much slower to compute. Multi dimensional (i.e volumes) FFT's are
+        decomposed so that each axis executes serially.
+
+        Parameters
+        ----------
+        output_scalars_name : str, optional
+            The name of the output scalars. By default this is the same as the
+            active scalars of the dataset.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        pyvista.UniformGrid
+            :class:`pyvista.UniformGrid` with the applied reverse FFT.
+
+        Examples
+        --------
+        Apply reverse FFT to an example image.
+
+        >>> from pyvista import examples
+        >>> image = examples.download_moonlanding_image()
+        >>> fft_image = image.fft()
+        >>> image_again = fft_image.rfft()
+        >>> image_again.point_data  # doctest:+SKIP
+        pyvista DataSetAttributes
+        Association     : POINT
+        Active Scalars  : PNGImage
+        Active Vectors  : None
+        Active Texture  : None
+        Active Normals  : None
+        Contains arrays :
+        PNGImage                float64  (298620, 2)          SCALARS
+
+        See :ref:`image_fft_example` for a full example using this filter.
+
+        """
+        self._check_fft_scalars()
+        alg = _vtk.vtkImageRFFT()
+        alg.SetInputDataObject(self)
+        _update_alg(alg, progress_bar, 'Performing Reverse Fast Fourier Transform.')
+        output = _get_output(alg)
+        self._change_fft_output_scalars(
+            output, self.point_data.active_scalars_name, output_scalars_name
+        )
+        return output
+
+    def low_pass(
+        self,
+        x_cutoff,
+        y_cutoff,
+        z_cutoff,
+        order=1,
+        output_scalars_name=None,
+        progress_bar=False,
+    ):
+        """Perform a butterworth low pass filter in the frequency domain.
+
+        This filter requires that the :class`UniformGrid` have complex point
+        scalars, usually generated after the :class:`UniformGrid` has been
+        converted to frequency domain by a :func:`UniformGridFilters.fft`
+        filter.
+
+        A :func:`UniformGridFilters.rfft` filter can be
+        used to convert the output back into the spatial
+        domain. This filter attenuates high frequency components.
+        Input and output are in doubles, with two components.
```

Comment:
Is this about two-component doubles still true?

Reply:
Fixed in 6ce17e5590254f7e030298d5bcb4c42cd0e46ca4